### PR TITLE
Improves the getting started comprehensive guide, expose services page

### PR DESCRIPTION
**Pull request by rspurgeon**
### Summary
Improves the getting started comprehensive guide, expose services page

    - Fixes some of the URLs, in particular the httpie ones were not consistent with curl
    - Added a blurb about Data Plane
    - Removed the `:` character from the port number reference

### Reason
Change based on my review during onboarding of this page. The Admin API reference at the end of the page was inaccurate. 

### Testing
Built the site locally and reviewed the content

### CODE REVIEW

File: app/gateway/2.8.x/get-started/comprehensive/expose-services.md [hunk 1]
```diff
@@ -288,17 +288,16 @@ A `201` message indicates the Route was created successfully.
...
-By default, {{site.base_gateway}} handles proxy requests on port `:8000`.
-
-From a web browser, enter `http://<admin-hostname>:8000/mock`.
+From a web browser, navigate to `http://<kong-gateway-host>:8000/mock/request`.
 
 {% endnavtab %}
-{% navtab Using the Admin API %}
 
-Using the Admin API, issue the following:
+{% navtab Using a command-line tool%}
```
Comment (lena-larionova) [inline app/gateway/2.8.x/get-started/comprehensive/expose-services.md:300]:
```suggestion
{% navtab Using the Admin API %}
```
Hold off on changing this. We have a lot of URL dependencies on this tab name, and it's currently a standard tab title we use across the docs, so we would have to do a sweep and change them all at once. If you think it should be changed, please file a ticket in DOCU and we'll take a look.

File: app/gateway/2.8.x/get-started/comprehensive/expose-services.md [hunk 2]
```diff
@@ -288,17 +288,16 @@ A `201` message indicates the Route was created successfully.
 
 ## Verify the Route is forwarding requests to the Service
 
+By default, {{site.base_gateway}} handles proxy requests on port `8000`. This is often referred to as the Data Plane.
```
Comment (lena-larionova) [inline app/gateway/2.8.x/get-started/comprehensive/expose-services.md:291]:
```suggestion
By default, {{site.base_gateway}} handles proxy requests on port `8000`. The proxy is often referred to as the data plane.
```

**Review by lena-larionova — CHANGES_REQUESTED** [review]

File: app/gateway/2.8.x/get-started/comprehensive/expose-services.md (hunk 1 above)
Comment (rspurgeon) [reply app/gateway/2.8.x/get-started/comprehensive/expose-services.md:300]: Ack, yes it's not accurate to say that it's the Admin API.  I will file a ticket

Comment (rspurgeon) [reply app/gateway/2.8.x/get-started/comprehensive/expose-services.md:300]: DOCU-2334

**Review by lena-larionova — APPROVED** [review]

**Pull request merged**